skimage/transform/_warps.py

```diff
@@ -178,6 +201,9 @@ def rescale(image, scale, order=1, mode=None, cval=0, clip=True,
     preserve_range : bool, optional
         Whether to keep the original range of values. Otherwise, the input
         image is converted according to the conventions of `img_as_float`.
+    multichannel : bool, optional
+        If True, last axis will not be rescaled.  The default is True for 3D
+        (2D+color) inputs, False otherwise.
```

Comment:
I'd rephrase it like this: ` By default, is set to True for 3D (2D+color) inputs, and False for others.`.